errai-security/errai-security-picketlink/src/main/java/org/jboss/errai/security/server/PicketLinkAuthenticationService.java

```diff
@@ -140,11 +140,10 @@ public User getUser() {
     Set<Role> roles = new HashSet<Role>();
 
     if (identity.isLoggedIn()) {
```

Comment:
below is the main change of this PR.
The main difference is that now it ask PicketLink if the user has a Role.